### ISSUE TITLE
Incredibles Quaternion bug

### ISSUE THREAD

**Issue by DashParrWare** [BassBoy320/io_anim_heavy_iron_studios#1]
![skb error](https://user-images.githubusercontent.com/107139901/186450165-e926ae1e-f3f5-4541-a9fd-9b08118fa1fc.png)

hi everytime i import the incredibles animations called skb1 and 1bks it gets wrong quaternions for some reason

**Comment by BassBoy320** [comment]
I'm working on it.

**Comment by BassBoy320** [comment]
Oops, accidentally closed it. I'm reopening...

**Comment by DashParrWare** [comment]
yeah some wrong lower joint rotations that i'm getting and most positions of the whole bones are sticking out by it's pivot bones, if only this will be fix of positions, the whole entire bones so ![dash improved abit](https://user-images.githubusercontent.com/107139901/186869996-5baed620-6707-44dc-8704-2bc06a34e5ea.png)
it will look like this, if you import it in
